ex8JQ/phonebook.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <link rel="stylesheet" href="style/style.css">
+  <script data-main="js/config" src="js/lib/require.js"></script>
+  <script type="application/javascript">
+    window.onload = function() {
+      require(['config'], function() {
+        require(['jquery'], function($) {
+          $(document).ready(function() {
+            require(['jquery','tmpl','ajaxGet'], function($, tmpl, ajaxGet) {
```

Comment:
You already have jquery. Why not move all these deps into the require statement above?